automatic/autohotkey.install/tools/chocolateyInstall.ps1

```diff
@@ -1,9 +1,34 @@
-﻿$name   = 'autohotkey.install'
-$url = '{{DownloadUrl}}'
-$url64 = $url
+﻿$ErrorActionPreference = 'Stop'
+
+$packageName = 'autohotkey.install'
+$url32       = 'https://autohotkey.com/download/1.1/AutoHotkey112401_Install.exe'
+$url64       = $url32
+$checksum32  = '78ea671c3305465933a5d00f39da7e86c116a8a5a1849835196a27799ac29f8f'
+$checksum64  = $checksum32
+
 $silent = '/S'
+if(Get-ProcessorBits 64) { $silent += ' /x64' }
+
+$packageArgs = @{
+  packageName            = $packageName
+  fileType               = 'EXE'
+  url                    = $url32
+  url64bit               = $url64
+  checksum               = $checksum32
+  checksum64             = $checksum64
+  checksumType           = 'sha256'
+  checksumType64         = 'sha256'
+  silentArgs             = $silent
+  validExitCodes         = @(0)
+  registryUninstallerKey = 'autohotkey'
+}
+Install-ChocolateyPackage @packageArgs
 
-$is64bit = (Get-WmiObject Win32_Processor).AddressWidth -eq 64
-if($is64bit) { $silent += ' /x64' }
+$installLocation = Get-AppInstallLocation $packageArgs.registryUninstallerKey
+if ($installLocation)  {
+    Write-Host "$packageName installed to '$installLocation'"

```

Comment:
I wouldn't do that. This code works almost always, while internal chocolatey stuff works maybe 50% of the time or less.

Also, the function lets u actually use install location while the choco reported one is unusable to the maintainer. People may use this in future to improve on updater, for example to associate some files to ahk (historically ahkl extension was manually added while AHK_L was considered alternative)